### PR TITLE
Initial attempt at known limitations doc

### PR DESCRIPTION
**Pull request by phobologic**
Any other ideas, go ahead and let me know (or update the doc).

### CODE REVIEW

File: docs/known_limitations.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Empire :: Known Limitations
+
+A good place to see a list of known issues and limitations is the
+[github issue list][github issue list]. Here's a list of well known
```
Comment (benguillet) [inline docs/known_limitations.md]: https://github.com/remind101/empire/issues ?

File: docs/known_limitations.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Right now Empire can only serve http & https services.
+
+## Only one exposed process per app
+
+You can only have a single exposed process per app, the `web` app.
+
+## Unable to update ELB for an app once it is deployed
+
+Due to a [bug][https://github.com/remind101/empire/issues/498] in
```
Comment (benguillet) [inline docs/known_limitations.md]: `[]()` no?

**Comment by ejholmes** [comment]
:+1:

**Comment by benguillet** [comment]
:+1:

**Pull request merged**